Formula/postgresql@11.rb

```diff
@@ -79,6 +79,11 @@ def post_install
 
   def caveats
     <<~EOS
+      Formula automatically runs `initdb` post-install, creating a database
+        named "postgres" and a Postgres superuser with an empty password
+        whose name is equal the name of the effective user executing the
```

Comment:
Is this a non-standard setting?

Reply:
I've been told by a Postgres dev that it is common, but there is no convention. Without carefully reading the formula code, it's not clear to users what exactly `brew` does when it installs. This is what bit me and what prompted the PR. 

Specifically, these appear to be variable in existing installers:

- username may be set to the current user or to a user named `postgres`
- Some installers prompt for a password, some default to empty, and others default to `postgres`

I found it very hard to find the right information here. I eventually read the formula code and found the docs on `initdb` and pieced it together. If you search for it, you'll find lots of guides and SO posts with conflicting information, and I'm certain this will be a cause for confusion for other Homebrew users.

Reply:
I think it's better to print the actual command run, then point them to the docs to find out more, so something like:
```
This formula has created a default database cluster with:
  initdb --locale=C -E UTF-8 #{var}/postgres
For more details, read:
  https://www.postgresql.org/docs/12/app-initdb.html
```
That way, the user would know which locale, encoding and cluster directory were chosen on their behalf. The rest of the details can be sussed out in the documentation, and if PostgreSQL subtly changes their defaults, the user isn't led astray by a stale caveat.

Reply:
Yeah, we can even use `#{version}` in the caveat to keep the link up to date.

In addition to that it might be good to add the more descriptive text to docs.brew.sh/FAQ. Caveats are pretty easy to miss, and can't be searched like docs.brew.sh can.

Reply:
Hi all - Would you like me to take a stab at this? I can make the change suggested by @gromgit, but I'm not sure how to incorporate those requested by @SMillerDev. And I suspect someone else will have to update the FAQ anyway?

Reply:
> I'm not sure how to incorporate those requested by @SMillerDev

If you mean:

> we can even use #{version} in the caveat to keep the link up to date

then he's saying to change my suggestion to this:
```
This formula has created a default database cluster with:
  initdb --locale=C -E UTF-8 #{var}/postgres
For more details, read:
  https://www.postgresql.org/docs/#{version.to_s.slice(/\d+/)}/app-initdb.html
```
which automagically substitutes the formula's major version number in the docs URL.

> And I suspect someone else will have to update the FAQ anyway?

Every page in docs.brew.sh can be forked, edited and PR'd. Just click on the _Fork me on GitHub_ banner on the top right, make your additions, then submit a PR.

That said, I think https://docs.brew.sh/Common-Issues may be a more suitable location, as it already contains questions about stuff Homebrew installs, while https://docs.brew.sh/FAQ is laser-focused on Homebrew itself.

Reply:
I updated the formula to reflect your suggestions. The [CI seems to be stuck](https://github.com/Homebrew/homebrew-core/pull/58085/checks?check_run_id=915241341) waiting to run one of the checks, however. Normally there is a button that allows you to re-run checks, but it doesn't appear for me here (I assume because I don't have access).

To me, both the FAQ and Common-Issues seem too general to warrant a Postgres formula-specifc blurb. But if you think otherwise and have suggestions for language, I don't mind getting a PR together.